UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/wwwroot/js/Custom/RadioNavigationWarning.js

```diff
@@ -0,0 +1,20 @@
+﻿$(document).ready(function () {
+
```

Comment:
Please add validation similar to other fields to make it consistent for user to understand